### ISSUE TITLE
help to send message/frame`

### ISSUE THREAD

**Issue by taoloo897** [pierremolinaro/acan-esp32#7]
Hi. Nice work here. Library works greate with esp32u -wroomer.
I have an issue
I want to create a message with id and data and send it with your library.
Please give me an example.
I have something like this 👍 
  tx_frame.FIR = CAN_frame_std;
  tx_frame.MsgID = 0x380;
  tx_frame..DLC = 4;
  tx_frame.data[0] = 0x32;
  tx_frame.data[1] = 0x00;
  tx_frame.data[2] = 0x55;
  tx_frame.data[3] = 0xFF;

And other issue if i want to receive frame and read data from it  (byte with byte), please, add and example too. 
Thanks

**Comment by taoloo897** [comment]
i made like this and will try to can magistral
frame.id= 0x38A;
  frame.len = 4;
  frame.ext = false;
  frame.rtr = false;
  frame.data[0] = b1;
  frame.data[1] = 0x00;
  frame.data[2] = b1;
  frame.data[3] = b4;
  ACAN_ESP32::can.tryToSend (frame);

**Comment by pierremolinaro** [comment]
Yes, you are wright. There is a PDF documentation in the extras directory of the library, and many sample codes in the examples directory.

**Comment by taoloo897** [comment]
Thanks a lot. I will open a new issue for esp32 -s3.
